test.sh

```diff
@@ -61,32 +82,44 @@ root() {
 
   echo "==> Running root level tests."
   set -e
-  go test $tags -timeout=25m . -v -race -parallel 16
+  go test $tags -v -race -parallel=16 -timeout=25m $covermode $coverprofile .
+  write_coverage
```

Comment:
you could do this in 1line, with an `&&`, I think that would look cleaner

Reply:
Done 👍